### PR TITLE
Default to processor count concurrent scan instead of 2 * processor count

### PR DESCRIPTION
**Pull request by crobibero**

### CODE REVIEW

**Comment by thornbill** [comment]
We will also need to update the placeholder value in the web settings.

File: MediaBrowser.Controller/Entities/Folder.cs [hunk 1]
```diff
@@ -593,8 +593,7 @@ void UpdateProgress()
                 progress.Report(childrenProgress.Average());
             }
 
-            var fanoutConcurrency = ConfigurationManager.Configuration.LibraryScanFanoutConcurrency;
-            var parallelism = fanoutConcurrency > 0 ? fanoutConcurrency : 2 * Environment.ProcessorCount;
+            var parallelism = Math.Max(ConfigurationManager.Configuration.LibraryScanFanoutConcurrency, 1);
```
Comment (thornbill) [inline MediaBrowser.Controller/Entities/Folder.cs:596]: This completely removes the ability to set the value to be based on processor count... is that desired? The validation and descriptions in web will need updated if so. 🙃

**Comment by cvium** [comment]
This isn't a 10.9 feature iirc. Isn't defaulting to 1 a step backwards?

File: MediaBrowser.Controller/Entities/Folder.cs (hunk 1 above)
Comment (crobibero) [reply MediaBrowser.Controller/Entities/Folder.cs:596]: Our database can't handle that much concurrency, so I think having it based on processor count won't actually work

**Comment by crobibero** [comment]
> This isn't a 10.9 feature iirc. Isn't defaulting to 1 a step backwards?

Before 10.9 it was defaulted to just ProcessorCount (https://github.com/jellyfin/jellyfin/commit/3982b0e057cc692875ba35573197b86abf8d7adb#diff-41e6711494c9c6ceb7d6aab39d16245177564b0711e402cb2c757d37c3b6815eL589)

We can change the default back to that if desired

**Comment by Bond-009** [comment]
> > This isn't a 10.9 feature iirc. Isn't defaulting to 1 a step backwards?
> 
> 
> 
> Before 10.9 it was defaulted to just ProcessorCount (https://github.com/jellyfin/jellyfin/commit/3982b0e057cc692875ba35573197b86abf8d7adb#diff-41e6711494c9c6ceb7d6aab39d16245177564b0711e402cb2c757d37c3b6815eL589)
> 
> 
> 
> We can change the default back to that if desired

Yes I think that's the correct solution here.
I changed it as it worked fine for me, but I only have 6 cores in my server and no SMT.

**Review by Shadowghost — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**